adagrams/game.py

```diff
@@ -1,13 +1,129 @@
 from random import randint
+from .constants import LETTER_POOL, SCORE_CHART, ALPHABET
 
 def draw_letters():
-    pass
+
+    alphabet_pool = dict(LETTER_POOL)
+    results = []
+
+    while len(results) < 10:
```

Comment:
It would be more descriptive to use a constant variable to reference 10 and also make code more self-documenting.

```suggestion
    NUM_TILES_ALLOWED_IN_HAND = 10
    while len(results) < NUM_TILES_ALLOWED_IN_HAND:
```